CHANGELOG.md

```diff
@@ -95,6 +95,8 @@
 * [CHANGE] Increase the allowed number of rule groups for small, medium_small, and extra_small user tiers by 20%. #11152
 * [CHANGE] Update rollout-operator version to 0.26.0. #11232
 * [CHANGE] Memcached: Set a timeout of `500ms` for the `ruler-storage` cache instead of the default `200ms`. #11231
+* [CHANGE] Memcached: Ignore initial DNS resolution failure, i.e. don't depend on Memcached on startup. #11602
```

Comment:
```suggestion
* [CHANGE] Memcached: Ignore initial DNS resolution failure, meaning don't depend on Memcached on startup. #11602
```

Reply:
Sure, good question! It's just better to always be as specific as possible and not use Latin abbreviations, as many people confuse their meanings. Here's a reference to Google's documentation style guide that supports this idea: https://developers.google.com/style/word-list